lib/jvm.sh

```diff
@@ -17,7 +17,7 @@ DEFAULT_JDK_16_VERSION="16.0.2"
 DEFAULT_JDK_18_VERSION="18.0.2.1"
 DEFAULT_JDK_19_VERSION="19.0.2"
 
-if [[ -n "${JDK_BASE_URL}" ]]; then
+if [[ -n "${JDK_BASE_URL:-}" ]]; then
```

Comment:
not shellcheck exactly, just unbound variable causing issues if I run this from a script context with `set -u`